python/nx-cugraph/nx_cugraph/tests/test_convert.py

```diff
@@ -88,7 +91,10 @@ def test_convert():
     cp.testing.assert_array_equal(Gcg.node_values["foo"], [10, 20])
     assert Gcg.edge_values == Gcg.edge_masks == {}
     H = nxcg.to_networkx(Gcg)
-    assert set(G.edges) == set(H.edges) == {(0, 1)}
+    if G.is_multigraph():
+        assert set(G.edges) == set(H.edges) == {(0, 1, 0)}
```

Comment:
I don't know if this is worth a comment or not, but can you explain why `G.edges` is used instead of `G.edges()`? (I'm not familiar with what the 3-tuple represents)

Reply:
nevermind, I see that the third value is an edge key from the docstring in MultiGraph.